test/integration/app-types/app-types.test.js

```diff
@@ -40,8 +40,8 @@ const appDir = __dirname
           ),
         ].map(([, line]) => +line)
 
-        const ST = 17
-        const ED = 34
+        const ST = 18
+        const ED = 35
```

Comment:
Not for your or this PR but this is a pretty wild assertion. Why did these numbers increase?

Reply:
Oh it's because we added the directive which shifted the location of each error by one

Reply:
yeah these aren't my favorite, but it is what it is 😅